src/business_logic/state/state_cache.rs

```diff
@@ -5,7 +5,7 @@ use std::collections::{HashMap, HashSet};
 
 /// (contract_address, key)
 // TODO: Change [u8; 32] to Felt.
-pub(crate) type StorageEntry = (Address, [u8; 32]);
+pub type StorageEntry = (Address, [u8; 32]);
```

Comment:
you can replace [u8; 32] with ClassHash

Reply:
mmmm I am not sure about this, 
although both are a [u8; 32], this is a storage key, not a ClassHash

Reply:
Done